app/models/collection_version.rb

```diff
@@ -40,6 +40,14 @@ def accessioned?
       DepositCollectionJob.perform_later(collection_version)
     end
 
+    after_transition on: :update_metadata do |collection_version, transition|
```

Comment:
Could you use `after_transition :new => :first_draft do ...`? Then we don't need the inner conditional

Reply:
Sure, I'll try it.